### ISSUE TITLE
Access to XMLHttpRequest has been blocked by CORS policy.

### ISSUE THREAD

**Issue by sskaditya** [bradleyg/django-s3direct#204]
I have raised a new issue cause the origin is not going to the s3 

https://github.com/bradleyg/django-s3direct/issues/168#issuecomment-619361901

**Comment by jsbls** [comment]
Have you tried checking the IAM access policies for your bucket? I don't think it's a CORS issue, rather an access one.

Also, try adding the region to the endpoint:
`AWS_S3_ENDPOINT_URL="https://s3-ap-south-1.amazonaws.com"`

**Comment by sskaditya** [comment]
Yes I have but still the same issue

**Comment by Shtaket** [comment]
I have same problem too, maybe anyone can have suggestions?

**Comment by mmcc5678** [comment]
I am getting intermittent CORS Failed issues too. I have reviewed the differences between those requests that pass and fail and cannot see a difference.

**Comment by jacopsd** [comment]
Try -for test purpose- in the CORSConfiguration:
 <AllowedOrigin>*</AllowedOrigin>

Also: the endpoint might be wrong. Do not use 's3', use 'glacier', e.g.:
AWS_S3_ENDPOINT_URL = 'glacier.eu-west-1.amazonaws.com'

Check this list of endpoints:
https://docs.aws.amazon.com/general/latest/gr/glacier-service.html
